### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ ReviewDeployPlan_Screenreader: Voiceover is not announcing the column header of table.

### ISSUE THREAD

**Issue by mspatil110** [microsoft/azuredatastudio#6945]
**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MAC iOS version : 10.16.4

**Additional Details:**
Tools Name: Accessibility insight tool
MAS References: MAS1.3.1, MAS4.1.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Provide all the details for Deploy Dacpac & hit next.
11. Navigate up to the table and  verify that voiceover is announcing the column header of table or not.

**Actual:**
Voiceover is not announcing the column header of table.

**Expected:**
Voiceover should announce  the column header of table.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will face difficulty to understand the table.

**MAS Reference:**

MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

MAS4.1.2- 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx?d=w248054a65e6847719e1e242fb5025730&csf=1&e=v1KF77

**Attachment ref:** 

![6945_Bug](https://user-images.githubusercontent.com/51433293/63671824-16d21c80-c7fe-11e9-92a4-dd83666416cc.png)
![6945_2_Bug](https://user-images.githubusercontent.com/51433293/63671864-349f8180-c7fe-11e9-97b0-31069e7d3352.png)

**Comment by kisantia** [comment]
The "You are currently on a cell, inside a table..." message only shows if you stay in the cell for a few seconds. On initially navigating to the header cells, the name and tooltip of the cell is read.
![image](https://user-images.githubusercontent.com/31145923/63898007-16778400-c9ac-11e9-93ba-f82e46832ff9.png)

![image](https://user-images.githubusercontent.com/31145923/63897993-fb0c7900-c9ab-11e9-972b-4eafeb7a5d93.png)
